cmd/brimcap/root/command.go

```diff
@@ -62,6 +64,19 @@ func (c *Command) Run(args []string) error {
 	return Brimcap.Exec(c, []string{"help"})
 }
 
+func (c *Command) AddRunnersToPath() error {
+	execpath, err := os.Executable()
+	if err != nil {
+		return err
+	}
+	dir := filepath.Dir(execpath)
+	pathEnv := os.Getenv("PATH")
+	zeekPath := filepath.Join(dir, "zeek")
+	suricataPath := filepath.Join(dir, "suricata")
+	pathEnv = strings.Join([]string{zeekPath, suricataPath, pathEnv}, string(os.PathListSeparator))
```

Comment:
If you put these at the end of PATH, then users can substitute their own versions.